common.go

```diff
@@ -51,6 +53,24 @@ const (
 	TLS_AES_256_CCM_8_SHA256     CipherSuite = 0x1305
 )
 
+func (c CipherSuite) String() string {
+	switch c {
+	case CIPHER_SUITE_UNKNOWN:
+		return "unknown"
+	case TLS_AES_128_GCM_SHA256:
+		return "TLS_AES_128_GCM_SHA256"
+	case TLS_AES_256_GCM_SHA384:
+		return "TLS_AES_256_GCM_SHA384"
+	case TLS_CHACHA20_POLY1305_SHA256:
+		return "TLS_CHACHA20_POLY1305_SHA256"
+	case TLS_AES_128_CCM_SHA256:
+		return "TLS_AES_128_CCM_SHA256"
+	case TLS_AES_256_CCM_8_SHA256:
+		return "TLS_AES_256_CCM_8_SHA256"
+	}
+	return fmt.Sprintf("invalid CipherSuite value")
```

Comment:
Did you mean to be passing back the value in this answer? If not, why are you bothering with Sprintf?

Reply:
That was the plan, but `%x` leads to an infinite recursion here. I'll find a workaround for that.

Reply:
Done.